requirements.txt

```diff
@@ -4,10 +4,12 @@ click
 croniter
 github3.py
 gitpython
+humanize
```

Comment:
I should probably remove this, as I created separate constraints and requirements files for build-summary